### DISCUSSION TITLE
MongoDB CDC读取存量数据时为什么不支持checkpoint

### DISCUSSION THREAD

**Discussion by DarthVagen** [apache/flink-cdc#2170] [Q&A]
Q&A中明确提到，MongoDB CDC Connector在读取存量数据阶段不做checkpoint，读取增量数据时才会做；但是看taskmanager的日志，debeziumSourceFunction其实是保存了source的offset的，即使读存量数据也会保存。为什么不能在错误重启时从这个offset开始？

**Comment by Jiabao-Sun** [comment]
使用增量快照读可以支持。

**Reply by DarthVagen** [reply]
感谢您的回复，请问增量快照读是指scan.incremental.snapshot.enabled这个选项吗？

**Reply by Jiabao-Sun** [reply]
对的
